apps/src/templates/Notification.jsx

```diff
@@ -8,37 +8,37 @@ const NotificationType = {
   information: 'information',
   success: 'success',
   failure: 'failure',
-  warning: 'warning'
+  warning: 'warning',
+  course: 'course'
 };
 
 const styles = {
   main: {
     borderWidth: 1,
     borderStyle: 'solid',
-    borderRadius: 3,
```

Comment:
I made some more dramatic changes to the (until then unused, afaik) Notification component in https://github.com/code-dot-org/code-dot-org/pull/15996.  Looks like I might need to parameterise those changes?